.github/workflows/npm-publish.yml

```diff
@@ -17,6 +17,7 @@ jobs:
           node-version: '16.x'
           registry-url: 'https://registry.npmjs.org/'
           scope: 'next-buefy'
+      - run: npm install
```

Comment:
@wesdevpro I think this line is no longer necessary, because we now include `package-lock.json`.